io/plugins/eu.esdihumboldt.hale.io.csv/src/eu/esdihumboldt/hale/io/csv/reader/internal/CSVInstanceCollection.java

```diff
@@ -85,8 +85,8 @@ private void proceedToNext() {
 
 			// initialize reader if necessary
 			if (csvReader == null) {
-				boolean skipFirst = reader.getParameter(CommonSchemaConstants.PARAM_SKIP_FIRST_LINE)
-						.as(Boolean.class, false);
+				int skipN = reader.getParameter(CommonSchemaConstants.PARAM_SKIP_N_LINES)
```

Comment:
I saw that although this is a different constant you use the same value `skip`. So this needs too be backwards compatible to to accepting boolean values (e.g. from cases where the old setting is passed via CLI or from projects saved with the old setting).

Reply:
Solved.

Reply:
I guess the PR has not been updated yet then?